json/src/utils.nr

```diff
@@ -87,28 +90,22 @@ pub fn json_tree_hasher<let N: u32>(
             stack[0] = [1, 0];
             tree_hashes[0] = [0, 0];
         }
+        */
```

Comment:
This part passes the test when commented out. Would you explain the reason you add this code?